### PR TITLE
Unit test for place service impl 1312

### PR DESCRIPTION
**Pull request by olenapetryshak**
https://github.com/ita-social-projects/GreenCity/issues/1312

### CODE REVIEW

**Review by antonbakalets — CHANGES_REQUESTED** [review]
Too many `any()`.

**Comment by olenapetryshak** [comment]
> Too many `any()`.

changes done

**Review by TarasDovganyuk — APPROVED** [review]

File: core/src/test/java/greencity/service/impl/PlaceServiceImplTest.java [hunk 1]
```diff
@@ -325,4 +307,138 @@ void findAllTest() {
...
+
+    @Test
+    void deleteByIdTest() {
+        when(placeRepo.findById(place.getId())).thenReturn(Optional.of(place));
+        when(placeRepo.save(place)).thenReturn(place);
+
+        placeService.deleteById(place.getId());
+
+        assertEquals(PlaceStatus.DELETED, place.getStatus());
+        verify(notificationService, times(0)).sendImmediatelyReport(any());
```
Comment (antonbakalets) [inline core/src/test/java/greencity/service/impl/PlaceServiceImplTest.java:338]: There is a more readable version 
`verify(notificationService, never()).sendImmediatelyReport(any());`

**Review by antonbakalets — APPROVED** [review]

File: core/src/test/java/greencity/service/impl/PlaceServiceImplTest.java (hunk 1 above)
Comment (olenapetryshak) [reply core/src/test/java/greencity/service/impl/PlaceServiceImplTest.java]: Done!

**Review by forestik — APPROVED** [review]

**Pull request merged**